### PR TITLE
make import without asterisk possible

### PR DESCRIPTION
**Pull request by RikSchefferOberon**
Currently react-native-keychain should be imported this way:
`import * as Keychain from 'react-native-keychain';`

I added an export default to be able to also import it this way:
`import Keychain from 'react-native-keychain';`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -271,3 +271,24 @@ export function setSharedWebCredentials(
...
+    getInternetCredentials,
+    resetInternetCredentials,
+    getOptionsArgument,
+    getMinimumSecurityLevel,
+    setGenericPassword,
+    getGenericPassword,
+    resetGenericPassword,
+    requestSharedWebCredentials,
+    setSharedWebCredentials
+}
```
Comment (friederbluemle) [inline index.js]: Please insert final newline at EOF.

**Comment by vonovak** [comment]
@RikSchefferOberon 

> Currently react-native-keychain should be imported this way:
> import * as Keychain from 'react-native-keychain';

personally, I'm a fan of named exports / imports - you can now import the functions also like this: `import { functionName } from 'react-native-keychain';` - does this not work well for your usecase?

**Comment by oblador** [comment]
I'm skeptical towards this implementation; it'll be a breaking change for all users to save you 4 characters. If we wanted to support it, then we might have a default export _in addition_, but not replace the current exports.

**Comment by oblador** [comment]
I was skeptical at first, but upon re-reading I think it could make sense to support both. The breaking change would be in that the `*` import will now also contain a `default` value, or what was your thinking @friederbluemle?

**Comment by RikSchefferOberon** [comment]
> I was skeptical at first, but upon re-reading I think it could make sense to support both. The breaking change would be in that the `*` import will now also contain a `default` value, or what was your thinking @friederbluemle?

Yes that is exactly my intention, to support both methods.

**Pull request merged**